### DISCUSSION TITLE
为什么ppstructure识别的时候，把不是表格的内容也识别成表格了？

### DISCUSSION THREAD

**Discussion by Huanuanx** [PaddlePaddle/PaddleOCR#12401] [Q&A] [answered]
table_max_len=488
layout_model_dir=picodet_lcnet_x1_0_fgd_layout_infer
table_model_dir=ch_ppstructure_mobile_v2.0_SLANet_infer

**Comment by crackso** [comment, marked as answer]
这个应该是分类模型的问题。分类模型识别不准确，导致表格区域错误。
